javamelody-core/src/main/java/net/bull/javamelody/internal/model/Collector.java

```diff
@@ -132,7 +139,12 @@ public Collector(String application, List<Counter> counters,
 			LOG.warn("exception while reading counters data from files in "
 					+ Parameters.getStorageDirectory(application), e);
 		}
-
+		try {
+			String strackTraceSizeParameter = Parameter.CURRENT_REQUEST_STACKTRACE_SIZE.getValue();
+			stackTraceSize =  strackTraceSizeParameter != null ? Integer.parseInt(strackTraceSizeParameter) : 1;
+		} catch (Exception exception) {
+			LOG.warn("Invalid stackTraceSize parameter: ", exception);

```

Comment:
invalid parameter exceptions are never catched, they are thrown to stop javamelody.